portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/PropertiesList.testcase

```diff
@@ -228,8 +224,6 @@ definition {
 
 			ACDXPSettings.connectAnalyticsCloud();
 
-			ACDXPSettings.navToSyncedSitesMenu();
-
 			ACUtils.assertItemsName(itemList = "Property OnlyAvailablePropertiesAppearInDXP 1");
 
 			ACUtils.assertItemsName(itemList = "Property OnlyAvailablePropertiesAppearInDXP 2");
```

Comment:
Hey @joshitagaki, I think I moved this test to the `ConnectionPropertyList.testcase` file and forgot to delete it in its source file. The test is fixed in the other file, so could you delete it in that file? Sorry for that!

Test Name: `OnlyAvailablePropertiesAppearInDXP`

Reply:
Ahh ok nice thanks for catching that. I'll delete it